changelog.md

```diff
@@ -272,6 +272,9 @@
 
 - Custom numeric literals (e.g. `-128'bignum`) are now supported.
 
+- Tuple expressions are now parsed consistently as
+  `nnkTupleConstr` node. Will affect macros expecting nodes to be of `nnkPar`.
```

Comment:
can you list here a representative set of cases that are changed from nnkPar to/from nnkTupleConstr ?

Reply:
I think that would be more confusing, since the old logic wasn't really consistent.

Reply:
An exhaustive list would be more of a documentation of the old inconsistent behaviour, rather than the new one, so I'm wary of doing that.